### DISCUSSION TITLE
Breadcumbar example

### DISCUSSION THREAD

**Discussion by jindal1979** [unoplatform/uno#11555] [General]
Is there an example available for breadcumbar for Mobile platforms like Android. Is it a stable control? 

I observed some issues in using breadcumbar and could not find any samples in Uno.

**Comment by jeromelaban** [comment]
The usage of breadcumbar is expected to be the same as [the ones specified here](https://learn.microsoft.com/en-us/windows/apps/design/controls/breadcrumbbar). If you found specific issues, let us know by providing specific samples for what's not working.

**Reply by jindal1979** [reply]
I will create a breadcumbar sample to demonstrate the issue. The issue is when there are many entries, the flyout shows 1 entry on 1st click and all entries on 2nd click.

**Reply by jindal1979** [reply]
Please find attached the sample application to demonstrate the bug in breadcumbar.  The issue is when there are many entries, the flyout shows 1 entry on 1st click and all entries on 2nd click.

I am also attaching the screen capture video. 

[BreadCumBarBug.zip](https://github.com/unoplatform/uno/files/10948287/BreadCumBarBug.zip)

https://user-images.githubusercontent.com/22019325/224479624-008f20a6-c8e2-44ff-aa40-194800c6738f.mp4

**Reply by jeromelaban** [reply]
@MartinZikmund does it look familiar?

**Reply by MartinZikmund** [reply]
Probably a items repeater issue, ~~can you open issue please?~~ Opened an issue, will look into it

**Reply by MartinZikmund** [reply]
@jindal1979 This should now be fixed!
